src/app/shared/components/navbar-mobile/navbar-mobile-dialog/navbar-mobile-dialog.component.html

```diff
@@ -1,13 +1,21 @@
 <mat-card class="navbar-mobile__dialog">
   <mat-card-header class="navbar-mobile__dialog__header">
-    <a (click)="this.startLogin()" *ngIf="!this.isAuthenticated; else userProfile" class="navbar__item" mat-button>Login</a>
+    <a (click)="this.startLogin()" *ngIf="!this.isAuthenticated; else userProfile" class="navbar__item navbar__user" mat-button>
```

Comment:
Quick question: do we replace `ngIf` now with `@if` as soon as we change a template that contains them? Or are we leaving them as they are?

Reply:
Good Question. I would say yes

Reply:
> _we change a template_ (...) _Or are we leaving them_

**Yes**
😂😂😂

But thanks for upgrading them to `@if/@else` 👍 I would have suggested this approach as well.